python/grass/temporal/__init__.py

```diff
@@ -1,33 +1,365 @@
+    "TemporalRasterAlgebraParser",
+    "TemporalRasterBaseAlgebraParser",
+    "TemporalTopologyDatasetConnector",
+    "TemporalVectorAlgebraLexer",
+    "TemporalVectorAlgebraParser",
+    "VectorAbsoluteTime",
+    "VectorBase",
+    "VectorDataset",
+    "VectorMetadata",
+    "VectorRelativeTime",
+    "VectorSTDSRegister",
+    "VectorSpatialExtent",
+    "adjust_datetime_to_granularity",
+    "aggregate_by_topology",
+    "aggregate_raster_maps",
+    "assign_valid_time_to_map",
+    "c_library_server",
+    "check_datetime_string",
+    "check_granularity_string",
+    "check_new_map_dataset",
+    "check_new_stds",
+    "collect_map_names",
+    "compute_absolute_time_granularity",
+    "compute_common_absolute_time_granularity",
+    "compute_common_absolute_time_granularity_simple",
+    "compute_common_relative_time_granularity",
+    "compute_datetime_delta",
+    "compute_relative_time_granularity",
+    "compute_univar_stats",
+    "count_temporal_topology_relationships",
+    "create_numeric_suffix",
+    "create_suffix_from_datetime",
+    "create_temporal_database",
+    "create_temporal_relation_sql_where_statement",
+    "create_time_suffix",
+    "dataset_factory",
+    "dataset_mapcalculator",
+    "datetime_to_grass_datetime_string",
+    "decrement_datetime_by_string",
+    "export_stds",
+    "extract_dataset",
+    "gcd",
+    "gcd_list",
+    "get_available_temporal_mapsets",
+    "get_current_gisdbase",
+    "get_current_location",
+    "get_current_mapset",
+    "get_database_info_string",
+    "get_dataset_list",
+    "get_enable_mapset_check",
+    "get_enable_timestamp_write",
+    "get_raise_on_error",
+    "get_sql_template_path",
+    "get_tgis_backend",
+    "get_tgis_c_library_interface",
+    "get_tgis_database",
+    "get_tgis_database_string",
+    "get_tgis_db_version",
+    "get_tgis_db_version_from_metadata",
+    "get_tgis_dbmi_paramstyle",
+    "get_tgis_message_interface",
+    "get_tgis_metadata",
+    "get_tgis_version",
+    "get_time_tuple_function",
+    "gran_plural_unit",
+    "gran_singular_unit",
+    "gran_to_gran",
+    "import_stds",
+    "increment_datetime_by_string",
+    "init",
+    "init_dbif",
+    "list_maps_of_stds",
+    "modify_datetime",
+    "modify_datetime_by_string",
+    "open_new_map_dataset",
+    "open_new_stds",
+    "open_old_stds",
+    "print_gridded_dataset_univar_statistics",
+    "print_spatio_temporal_topology_relationships",
+    "print_temporal_topology_relationships",
+    "print_vector_dataset_univar_statistics",
+    "profile_function",
+    "register_map_object_list",
+    "register_maps_in_space_time_dataset",
+    "relative_time_to_time_delta",
+    "relative_time_to_time_delta_seconds",
+    "run_mapcalc2d",
+    "run_mapcalc3d",
+    "run_vector_extraction",
+    "sample_stds_by_stds_topology",
+    "set_raise_on_error",
+    "set_spatial_relationship",
+    "set_temoral_relationship",
+    "set_temporal_relationship",
```

Comment:
```suggestion
    "set_temoral_relationship",
    "set_temporal_relationship",
```
"set_temporal_relationship" doesn't exist as caught by pylint E0603 and pylance, verified with a text search. But "set_temoral_relationship" is present, and is obviously wrong.

Is renaming `set_temoral_relationship` a breaking change? It's been there for like 12 years..

Search of https://github.com/search?q=set_temoral_relationship+NOT+is%3Afork&type=code and https://sourcegraph.com/search?q=context:global+set_temoral_relationship&patternType=keyword&sm=0 seems that there is no public external usage otherwhere, nor addons

Reply:
I searched through the grass repo and I did not see it being used anywhere

Reply:
There is no usage of this function so imo it should be good to rename as well

Reply:
I feel like fixing a typo in the function name, without apparent usage, even if it isn't inside gui, can be done. It would be a change reflected in a yearly release, (8.5), so it seems an appropriate impact size

Reply:
I have made the update replacing `temoral` with `temporal`